packages/eslint-plugin-svelte/tests/fixtures/rules/no-at-debug-tags/invalid/debug01-output.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
```

Comment:
There is no `{@debug}` inside, does it been remove by linting fix itself?

Reply:
this is the test output, i.e. after it has passed through the fixer. so it shouldn't be there

generally complex changes are suggestions, and simple ones are fixes. happy to make this a suggestion if people want though